faq/how_can_i_extend_the_reading_functions_with_a_new_dataformat.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
please remove the corresponding page with the incorrect name

Reply:
as far as I see the old page with incorrect name was already removed (from the faq folder). where else would i remove it from?

Reply:
I checked out your repository and switched to the `faqtypos` branch. The file 
`faq/how_can_extend_the_reading_functions_with_a_new_dataformat.md` is still there, alongside `faq/how_can__i_extend_the_reading_functions_with_a_new_dataformat.md`.

Also, if you go to https://github.com/fieldtrip/website/pull/188/files you can see that the original file with the incorrect name has not been deleted (otherwise it would show up as "changed").

Reply:
the mistake I made was that I had not staged the changes before commit. Hope it's good now.